api/src/extensions/manager.ts

```diff
@@ -1,5 +1,14 @@
 import { useEnv } from '@directus/env';
+import { APP_SHARED_DEPS } from '@directus/extensions';
+import { HYBRID_EXTENSION_TYPES } from '@directus/constants';
```

Comment:
```suggestion
import { APP_SHARED_DEPS, HYBRID_EXTENSION_TYPES } from '@directus/constants';
```

Reply:
@ComfortablyCoding I didn't move over `APP_SHARED_DEPS` to `@directus/constants`. I am happy to do that, but it was not needed to get the types working for services.

Reply:
No that is fine, I thought I saw it was moved.